src/main/java/ingredientes/Base.java

```diff
@@ -26,7 +26,7 @@ public boolean equals(Object o) {
     //É necessário consertar o compareTo, para imprimir na ordem correta.
     @Override
     public int compareTo(Ingrediente ingrediente) {
-        return ingrediente.obterTipo().toString().compareToIgnoreCase(this.obterTipo().toString());
+        return ingrediente.obterTipo().toString().compareToIgnoreCase(this.obterTipo().toString()) * -1;
```

Comment:
o return ficou invertido, era necessário arrumar:

return **this**.obterTipo().toString().compareToIgnoreCase(**ingrediente**.obterTipo().toString());